src/nupic/utils/LoggingException.cpp

```diff
@@ -34,7 +34,11 @@ LoggingException::~LoggingException() throw()
   if (!alreadyLogged_) {
     // Let LogItem do the work for us. This code is a bit complex
     // because LogItem was designed to be used from a logging macro
+#if defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC) && defined(NDEBUG)
+    auto li = new LogItem(filename_.c_str(), lineno_, LogItem::info);
```

Comment:
Actually I noticed in AppVeyor there is a "Messages" tab at the top of the console output that shows all of the stderr output. It looks like it doesn't like the stderr messages that get written during testing. I think a temporary solution is to redirect stderr to a file when you execute the tests in AppVeyor. But we will need to make sure that AppVeyor still recognizes a real error if we do that.